packages/puppeteer-core/package.json

```diff
@@ -83,8 +83,8 @@
       ],
       "clean": false,
       "files": [
-        "lib/esm/third_party/**",
-        "lib/cjs/third_party/**"
+        "third_party/**",
```

Comment:
@jrandolf Any specific reason we had it set-up like so? It was causing a circular update when running watch mode.

Reply:
This is related to how `wireit` works. It's a little finicky, but if this works for us, then I'm down for the change.

Reply:
Will take a look after build failures are fixed